livekit_egress.proto

```diff
@@ -83,6 +84,19 @@ message WebEgressRequest {
   repeated SegmentedFileOutput segment_outputs = 11;
 }
 
+// record audio and video from a single participant
+message ParticipantEgressRequest {
+  string room_name = 1; // required
+  string identity = 2;  // required
```

Comment:
would be great to have `TrackSource source`, defaulting to Camera, but can be ScreenShare